tests/input/test_slider.py

```diff
@@ -0,0 +1,75 @@
+import random
+from typing import Any, Callable, Dict, Optional
+
+import pytest
+from dash import Dash, Input, Output
+from dash.testing.composite import DashComposite
+from selenium.webdriver.common.action_chains import ActionChains
+
+import dash_mdc_neptune as mdc
+
+
+SLIDER_OPTIONS = {"minValue": 0, "maxValue": 100, "stepValue": 1, "selected": 50, "width": 100}
+
+
+@pytest.fixture
+def dash_app() -> Callable[[Optional[int]], Dash]:
+    def app_factory(selected: Optional[int] = None) -> Dash:
+        kwargs: Dict[str, Any] = SLIDER_OPTIONS
+        if selected is not None:
+            kwargs["selected"] = selected
+
+        app = Dash(name=__name__)
+        app.layout = mdc.Dashboard(
+            children=mdc.Page(
+                children=mdc.Section(
+                    children=mdc.Box(
+                        children=[
+                            mdc.Slider(id="slider", **kwargs),
+                            mdc.Typography(id="text"),
+                        ],
+                    ),
+                    cards=[{"title": "Card"}],
+                ),
+            )
+        )
+
+        @app.callback(
+            Output(component_id="text", component_property="text"),
+            Input(component_id="slider", component_property="selected"),
+        )
+        def on_change(selected: int) -> str:
+            return str(selected)
+
+        return app
+
+    return app_factory
+
+
+@pytest.mark.parametrize(
+    "selected,text",
+    [
+        (0, "0"),
+        (50, "50"),
+        (100, "100"),
+    ],
+)
+def test_props(dash_duo: DashComposite, dash_app, selected, text):
```

Comment:
❓ 

Out of curiosity, why has a typing on the Dash `pytest` fixture (`dash_duo`) been added onto the test signature function?

Reply:
To have code completion/discovery when writing the test as I was unaware what functions were available. Do you want me to remove it?